res/controllers/midi-components-0.0.js

```diff
@@ -318,21 +318,68 @@
         this.firstValueReceived = false;
     };
     Pot.prototype = new Component({
+        relative: false,
         inValueScale: function (value) { return value / this.max; },
-        input: function (channel, control, value, status, group) {
-            this.inSetParameter(this.inValueScale(value));
-            if (! this.firstValueReceived) {
-                this.firstValueReceived = true;
-                this.connect();
+        shift: function () {
+            if (this.relative) {
+                this.input = function (channel, control, value, status, group) {
```

Comment:
Ah thank you. Maybe some comments in the code could be useful. 

Lets verify the current behavior: 

This means If I set the rate slider to +-6 % I can move to the very top = -6 %, press shift and move to the bottom, still on -6 % now I can reach -12 % on the very top. 
Is this a bug or a feature? 

With non out of bounds controls I can move to each end and I am sure the relative slider is centered again. 

For now I do not see big use-cases for the first one, but the second seams to be handy. 
What is your use-case for the first?